content/docs-v1/figure-images.md

```diff
@@ -20,117 +18,93 @@ Quire does not require a specific image file format or size, but we have some re
 
 {{< q-class "box tip" >}}
 
-- Some themes may offer additional options, and styles may be edited and new styles added in any theme with {{< q-def "CSS" >}}.
+- If you would like to label your figure image add a `label` attribute to the image entry in your  `figures.yaml` file.
 
 {{< /q-class >}}
 
-## Create and Style Figure Groups with q-figure-group Shortcode
+## Style Figure Images
 
-If your project uses a `figures.yaml` file, you can also create a group of figures by using the `q-figure-group` shortcode and simply including multiple, comma-separated values in the `id` field.
+By default, figures will display at the width of the full-column of text. Modifier classes can be added to a shortcode to style the way the figures appear. Available classes are `is-pulled-left` and `is-pulled-right`. This will put the image in-line with the text, either to the right or left. Within the shortcode, the first value is the `id` and the second value is the class type. They should both appear in quotes, separated by a comma.
 
-```go
-{{</* q-figure-group id="1.1, 1.2" */>}}
+```md
+{% figure "1.2", "is-pulled-left" %}
 ```
 
-In the above example, each figure’s caption will be included in the grouping. Alternatively, if you add a `caption` attribute directly in the shortcode, it will override those present in the `figures.yaml` file and display with the group alone as a single, group caption.
+{{< q-figure id="1.9" >}}
 
-{{< q-figure id="1.10" >}}
+## Create and Style Figure Groups with the {% figure-group %} Shortcode
 
-Just as with the single `q-figure` shortcode, classes can be added to groups to style them. For example, to create a small group of images running along one side of your text.
+If your project uses a `figures.yaml` file, you can also create groupings of figures by using the `{% figure-group %}` shortcode. Within the shortcode, the `ids` of the images you would like included should appear separated by commas within one set of quotes.
 
-```go
-{{</* q-figure-group class="is-pulled-left" id="1.1, 1.2" */>}}
+```md
+{% figure-group "1.1, 1.2" %}
 ```
 
-In addition to all the attributes available to the `q-figure` shortcode, the `q-figure-group` extension also supports the `grid` attribute to specify a preferred grid width. In the below example, a `grid="2"` is specified and so the gallery grid will be 2 images wide at your publication layout’s full-size. Alternately, if you specified `grid="4"` the grid would be 4 images wide, making each image relatively smaller.
+The `{% figure-group %}` shortcode additionally supports the ability to specify a preferred grid width. In the below example, the inclusion of `"2"` before the list of `ids` means that the image grid will be 2 images wide at your publication layout’s full-size. Alternately, if you specified `"4"` the grid would be 4 images wide, making each image relatively smaller.
```

Comment:
The grid value is required currently (though I have the notion of maybe giving all figuregroups a grid of 2 or 3 by default and then letting people customize that using a class like `'is-grid-4'` or something. so we have a more consistent use of classes to control styling/layout things that people can get used to. We'll see though.)